cmd/kubeadm/app/phases/selfhosting/selfhosting.go

```diff
@@ -21,7 +21,8 @@ import (
 	"os"
 	"time"
 
-	apps "k8s.io/api/apps/v1"
+	"k8s.io/api/apps"
```

Comment:
You are adding references to the internal version where previously there weren't any? I am not sure what the purpose of this change is, but I expected it to be to remove references to the internal version. People should get the constants from the external version.

Reply:
Hi @lavalamp , I just consolidate the versioned constant in staging/src/k8s.io/api/apps/v1, staging/src/k8s.io/api/apps/v1beta1 and staging/src/k8s.io/api/apps/v1beta2 to a global file named const.go in staging/src/k8s.io/api/apps, I think people still get the constants from the external version. Otherwise I removed the internal versioned constant.

Reply:
I agree with @lavalamp - this is making consumers pull in the internal version when it should not be.

Reply:
`staging/src/k8s.io/api/apps/consts` is way more palatable as a location. I think that location is at best only slightly better than the current state of affairs, though, and so I'm not sure this change is worth the churn for us or for our users.

Reply:
> staging/src/k8s.io/api/apps/consts is way more palatable as a location. I think that location is at best only slightly better than the current state of affairs, though, and so I'm not sure this change is worth the churn for us or for our users.

I agree this is not worthwhile.